### PR TITLE
Added Mako Framework for PHP [ci fw PHP/mako]

### PR DESCRIPTION
**Pull request by n0nag0n**
Adding another PHP framework. Please and thank you!

### CODE REVIEW

File: frameworks/PHP/mako/benchmark_config.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "fortune_url": "/fortunes",
+      "update_url": "/update?queries=",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Fullstack",
+      "database": "MySQL",
+      "framework": "mako",
+      "language": "PHP",
+      "flavor": "PHP8",
+      "orm": "raw",
```
Comment (joanhey) [inline frameworks/PHP/mako/benchmark_config.json:18]: I think that it's using an ORM. It isn't using SQL direct.
I don't know if it is Micro or Full

**Review by joanhey — CHANGES_REQUESTED** [review]

**Comment by joanhey** [comment]
We are moving all the PHP frameworks to PHP 8.2.
If they work with v 8.2, please update to use it.

File: frameworks/PHP/mako/mako.dockerfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:22.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -yqq && apt-get install -yqq software-properties-common > /dev/null
+RUN LC_ALL=C.UTF-8 add-apt-repository ppa:ondrej/php > /dev/null && \
+    apt-get update -yqq > /dev/null && apt-get upgrade -yqq > /dev/null
+
+RUN apt-get install -yqq nginx git unzip \
```
Comment (joanhey) [inline frameworks/PHP/mako/mako.dockerfile:9]: I tested to have faster builds. But it have problems.
Please udpate to:
`RUN apt-get update -yqq > /dev/null && apt-get install -yqq nginx git unzip \`

**Review by joanhey — CHANGES_REQUESTED** [review]

File: frameworks/PHP/mako/mako.dockerfile (hunk 2 above)
Comment (n0nag0n) [reply frameworks/PHP/mako/mako.dockerfile:9]: the `apt-get update` line is on line 6, are you wanting me to remove that or keep both?

File: frameworks/PHP/mako/benchmark_config.json (hunk 1 above)
Comment (n0nag0n) [reply frameworks/PHP/mako/benchmark_config.json]: It's full. My bad. I copied this from Phalcon and even though they have a Full ORM, they are using "raw" :'(

**Comment by n0nag0n** [comment]
Updated the benchmark config and updated to php 8.2. I'll test my other frameworks I submitted to see if they work on php 8.2 as well.

**Comment by joanhey** [comment]
#7703

**Pull request merged**